qualification/pytest-jenkins.ini

```diff
@@ -11,8 +11,8 @@ use_ibv = true
 log_cli = true
 log_cli_level = info
 addopts = --report-log=report.json
-default_antennas = 8
-max_antennas = 10
+default_antennas = 26
+max_antennas = 40
```

Comment:
If you're wondering why I haven't used 40 for both: antenna-channelised-voltage tests use a dsim per antenna, so 26 is as much as the system can handle right now. The other tests use only a single dsim.